### PR TITLE
docs: fixed typos

### PR DESCRIPTION
**Pull request by Lioness100**
Is substraction a word?

### CODE REVIEW

**Comment by favna** [comment]
> Is substraction a word?

No the proper spelling is `subtraction`

File: packages/time-utilities/src/lib/DurationFormatter.ts [hunk 1]
```diff
@@ -35,10 +35,10 @@ export class DurationFormatter {
 		if (negative) duration *= -1;
 
 		for (const [type, timeDuration] of kTimeDurations) {
-			const substraction = duration / timeDuration;
-			if (substraction < 1) continue;
+			const subtraction = duration / timeDuration;
```
Comment (favna) [inline packages/time-utilities/src/lib/DurationFormatter.ts:38]:
```suggestion
			const division = duration / timeDuration;
```

it's a division after all

File: packages/time-utilities/src/lib/DurationFormatter.ts [hunk 2]
```diff
@@ -35,10 +35,10 @@ export class DurationFormatter {
...
+			if (subtraction < 1) continue;
```
Comment (favna) [inline packages/time-utilities/src/lib/DurationFormatter.ts:39]:
```suggestion
			if (division < 1) continue;
```

File: packages/time-utilities/src/lib/DurationFormatter.ts [hunk 3]
```diff
@@ -35,10 +35,10 @@ export class DurationFormatter {
...
 
-			const floored = Math.floor(substraction);
+			const floored = Math.floor(subtraction);
```
Comment (favna) [inline packages/time-utilities/src/lib/DurationFormatter.ts:41]:
```suggestion
			const floored = Math.floor(division);
```

**Review by favna — CHANGES_REQUESTED** [review]

File: packages/time-utilities/src/lib/DurationFormatter.ts (hunk 1 above)
Comment (Lioness100) [reply packages/time-utilities/src/lib/DurationFormatter.ts]: yeah idk where substraction came from lol

**Review by favna — APPROVED** [review]

**Pull request merged**